Packs/IP-API/Integrations/IPAPI/IPAPI.py

```diff
@@ -0,0 +1,99 @@
+import traceback
+from typing import Any, Dict, List
+
+import demistomock as demisto  # noqa: F401
+import urllib3
+from CommonServerPython import *  # noqa: F401
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    def get_ip_reputation(self, ip: str) -> Dict[str, Any]:
+        if demisto.params().get('https'):
+            return self._http_request(
+                method='GET',
+                url_suffix=ip,
+                params={
+                    'key': demisto.params().get('apikey'),
+                    'fields': 536608767
```

Comment:
Please remove this line - and replace with:
'fields': demisto.params().get('fields')